src/Storages/S3Queue/StorageS3Queue.cpp

```diff
@@ -465,6 +470,15 @@ std::shared_ptr<StorageS3Queue::FileIterator> StorageS3Queue::createFileIterator
     return std::make_shared<FileIterator>(files_metadata, std::move(glob_iterator), shutdown_called);
 }
 
+BlobStorageLogWriter StorageS3Queue::getBlobStorageLog()
```

Comment:
May be put this function into some Common folder? To avoid copying it in several places, it anyway does not use any class member

Reply:
Put it into `BlobStorageLogWriter` as static function